homeassistant/components/influxdb.py

```diff
@@ -153,7 +159,9 @@ def influx_event_listener(event):
         ]
 
         for key, value in state.attributes.items():
-            if key != 'unit_of_measurement':
+            if key in tags_attributes:
+                json_body[0]['tags'][key] = value
```

Comment:
Yeah, but at least previously some types have still lead to problems. See https://github.com/home-assistant/home-assistant/issues/3657. At that time the influx library in use didn't handle/convert some types good. I'm not sure of the current state. Anyway, in this case the user can choose what attribute to record, so I don't think it will be a problem.